test/test_cli.rb

```diff
@@ -137,6 +137,7 @@ def test_control_clustered
     s.close
 
     assert_match(/\{"started_at":"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z","workers":2,"phase":0,"booted_workers":2,"old_workers":0,"worker_status":\[\{"started_at":"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z","pid":\d+,"index":0,"phase":0,"booted":true,"last_checkin":"[^"]+","last_status":\{"backlog":0,"running":2,"pool_capacity":2,"max_threads":2,"requests_count":0\}\},\{"started_at":"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z","pid":\d+,"index":1,"phase":0,"booted":true,"last_checkin":"[^"]+","last_status":\{"backlog":0,"running":2,"pool_capacity":2,"max_threads":2,"requests_count":0\}\}\]\}/, body.split("\r\n").last)
+    assert_equal(Puma.stats, JSON.parse(body.split("\r\n").last, symbolize_names: true))
```

Comment:
Same here, redundant test.

Reply:
Maybe we can test Puma.stats output separately in another test if we don't do that explicitly anywhere else?

Reply:
Yeah, I added these lines because my first iteration of this PR unintentionally changed keys in `Puma.stats` from symbols to strings (`JSON.parse` default) without failing any tests, so I thought that behavior should be covered by an extra assertion somewhere. I can move it to a new test to make it more clear.